> > E' possibile mergiarla?
> 
> Coi test rossi, no.

Buongiorno, per entrambe i test falliti l'errore è 

2025-02-10 14:40:54,415 534 ERROR odoo odoo.addons.l10n_it_account_stamp.tests.test_account_stamp_invoicing: FAIL: InvoicingTest.test_tax_stamp_line_button
Traceback (most recent call last):
  File "/__w/l10n-italy/l10n-italy/l10n_it_account_stamp/tests/test_account_stamp_invoicing.py", line 117, in test_tax_stamp_line_button
    self.assertTrue(invoice.tax_stamp)
AssertionError: False is not true

non è dovuto a questa pr in particolare

--- a/l10n_it_delivery_note/models/stock_delivery_note.py
+++ b/l10n_it_delivery_note/models/stock_delivery_note.py
@@ -609,7 +609,6 @@ class StockDeliveryNote(models.Model):
                 note.date = datetime.date.today()
 
             if not note.name:
-                note.name = sequence.next_by_id()
                 # Avoid duplicates
                 while True:
                     name = sequence.with_context(

